neo/test/rawiotest/test_plexon2rawio.py

```diff
@@ -25,8 +26,45 @@ class TestPlexon2RawIO(
 ):
     rawioclass = Plexon2RawIO
     entities_to_download = ["plexon"]
-    entities_to_test = ["plexon/4chDemoPL2.pl2"]
+    entities_to_test = ["plexon/4chDemoPL2.pl2",
+                        "plexon/NC16FPSPKEVT_1m.pl2"
+                        ]
+    
 
+    def test_check_enabled_flags(self):
+        """
+        This test loads a 1-minute PL2 file with 16 channels' each
+        of field potential (FP), and spike (SPK) data. The channels 
+        cycle through 4 possible combinations of m_ChannelEnabled 
+        and m_ChannelRecordingEnabled - (True, True), (True, False), 
+        (False, True), and (False, False). With 16 channels for each 
```

Comment:
How can a channel be not enabled but recorded ?

Reply:
https://github.com/NeuralEnsemble/python-neo/pull/1546#issue-2505718693

See comment here Sam :)